### PR TITLE
add signalr.js v1.0.0 with git auto-update

### LINKED ISSUE #658: SignalR
Hi,

I'd like you to host signalr js library, from microsoft.

thanks

### PR DESCRIPTION
**Pull request by CreativeColon**
Resolves #658, #5934, #5878

Contains compiled and minified version of signalr.js v1.0.0 built with `build.ps1` script provided by official repository [here] (https://github.com/SignalR/SignalR/tree/77cd25394efa82901ab7061a014aa39b9f27e0c8/src/Microsoft.AspNet.SignalR.Client.JS)

### CODE REVIEW

File: ajax/libs/signalr.js/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "signalr.js",
+  "version": "1.0.0",
+  "filename": "jquery.signalR.js",
```
Comment (PeterDaveHello) [inline ajax/libs/signalr.js/package.json]: Please point to the minified file.

File: ajax/libs/signalr.js/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/CreativeColon/signalR.js.git"
+  },
+  "dependencies": {
+    "jquery": "^1.6.4"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/CreativeColon/signalR.js.git",
```
Comment (PeterDaveHello) [inline ajax/libs/signalr.js/package.json]: Auto-update should rely on official source.

Comment (CreativeColon) [inline ajax/libs/signalr.js/package.json]: The official source does not contain compiled files. That's the reason I've created this repo

**Comment by CreativeColon** [comment]
@PeterDaveHello  The official source does not contain compiled files. That's the reason I've created this repo

**Comment by PeterDaveHello** [comment]
@CreativeColon I know, but this makes no sense.

**Comment by CreativeColon** [comment]
@PeterDaveHello  So, there's no way to host signalr.js unless the official repo supports it by providing the script files ?

**Comment by PeterDaveHello** [comment]
@CreativeColon I think i can build it, will take a look at it later.

**Comment by CreativeColon** [comment]
@PeterDaveHello Ok, later then

**Comment by PeterDaveHello** [comment]
@CreativeColon What about you just remove the auto-update config and then I merge this PR? So that you still get the credit!

**Comment by CreativeColon** [comment]
@PeterDaveHello Yes, I will do that. But what about updating the versions should I do it manually and make a pull request for every release

**Comment by PeterDaveHello** [comment]
Yes, it'll be needed.

**Comment by CreativeColon** [comment]
@PeterDaveHello Ok, I've removed the auto-update block and `package.json` now points to the minified file.

Just as a heads-up the current version of SignalR is **2.2.0**. So, I'll have to do **multiple pull requests** to get there or a **single pull request** to update the repository from **1.0.0** to **2.2.0** and then make subsequent pull requests for every release from there. What would you prefer ?

**Comment by PeterDaveHello** [comment]
Single is fine, and please squash them into only one commit, thanks!

**Comment by CreativeColon** [comment]
Versions from v1.0.0 to v2.2.0 (latest stable release) are in place. Sorry, it took another commit to fix the package.json format.

**Comment by PeterDaveHello** [comment]
Nope, should squash them into one commit, thanks.

**Comment by CreativeColon** [comment]
Yes, I've squashed them to a single commit now.

**Comment by PeterDaveHello** [comment]
Nope there are three, and also, please rebase this branch with our latest master branch, thank you.

**Comment by CreativeColon** [comment]
@PeterDaveHello I've squashed them into a single commit after rebasing from the latest master branch.

**Comment by PeterDaveHello** [comment]
Thanks!

**Pull request merged**